cmd/rook/ceph/ceph.go

```diff
@@ -54,7 +54,8 @@ type config struct {
 }
 
 func init() {
-	Cmd.AddCommand(operatorCmd,
+	Cmd.AddCommand(cleanUpCmd,
+		operatorCmd,
```

Comment:
Why do we need a new CLI arg for this? Can't we just use operator code for this? `cephv1.CephCluster` gives you everything you need right?

Reply:
If we were only cleaning the host path I could easily see this as a script that just calls `rm -fr`. The tricky thing will be when devices are cleaned up. I think we'll need code that detects devices that had an OSD so we know which devices to wipe.

Reply:
Wouldn't the device wipe be a different job? I still think we can make it without the new CLI arg.

Reply:
So the same CLI call will try to delete everything/detect if mon or host?

Reply:
Yes. it will delete the cluster namespace in the dataDirHostPath and also delete the `mon-a` etc directories.  For example, 
```
[vagrant@worker0 rook]$ ls
mon-a  rook-ceph 
```
It will delete `rook-ceph` and `mon-a` directories.

Reply:
I'm still not certain this is necessary when a bash script in the cleanup spec is all that's necessary.

```bash
#!/usr/bin/bash
set -eEuo pipefail

rm -rf $DATA_DIR_HOST_PATH/rook-ceph
rm -rf $DATA_DIR_HOST_PATH/mon-*
```

Reply:
We actually discussed in huddle today two cases where we would need code:
1. To delete the mon dirs we want to guarantee the mons belong to the same cluster. So we need to analyze the mon dir and check if it has the same keyring before we delete it.
1. When we cleanup devices (future PR), it will also require some analysis directly on the node to determine which devices to clean up. The operator doesn't know details about devices in use on each node.

Reply:
Is that actually an issue? Shouldn't clusters be using different `dataDirHostPaths` so they don't collide?